src/app/records/Records.tsx

```diff
@@ -87,7 +88,8 @@ export default function Records({ account }: Props) {
 
   // We discover the rest of the root nodes from the Bungie.net core settings
   const otherHashes = destiny2CoreSettings
-    ? objectKeys(destiny2CoreSettings)
+    ? // https://github.com/microsoft/TypeScript/issues/15300#issuecomment-1320421641
+      objectKeys(destiny2CoreSettings as Pick<Destiny2CoreSettings, keyof Destiny2CoreSettings>)
         .filter((k) => k.includes('RootNode') && k !== 'craftingRootNodeHash')
         .map((k) => destiny2CoreSettings[k] as number)
     : [];
```

Comment:
Maybe this could be
```ts
const otherHashes = destiny2CoreSettings
    ? _.compact(
        Object.entries(destiny2CoreSettings).map(
          ([key, value]) =>
            key.includes('RootNode') && key !== 'craftingRootNodeHash' && (value as number)
        )
      )
    : [];
```

I don't like `_.compact(_.map)` and would like something like [`filterMap`](https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.filter_map) but I feel like this is better typed...